dipy/data/fetcher.py

```diff
@@ -592,16 +595,19 @@ def read_syn_data():
 """
 
 
-def read_mni_template(contrast="T2"):
+def read_mni_template(version="a", contrast="T1"):
```

Comment:
Well, I personally think it's better for backwards compatibility to keep the old value.

Also, which template was fetched before the 2 versions were added? Asking because I just want to know if setting ```version="a"``` will keep the same behavior as before for default cases.